airbyte-integrations/connectors/source-google-webfonts/source_google_webfonts/schemas/fonts.json

```diff
@@ -0,0 +1,113 @@
+{
```

Comment:
the indentation in this json file is a little off, can you adjust it? Thanks.

Reply:
Done